Specs/Braintree-Acceptance-Specs/BTDropInAcceptanceSpec.m

```diff
@@ -118,6 +141,78 @@
         });
     });
 
+    fdescribe(@"card form", ^{
```

Comment:
Probably don't want the `f` here. But when I removed it, I found that other test(s) don't pass